bin/pact-cli.spec.ts

```diff
@@ -3,11 +3,11 @@ import chaiAsPromised = require("chai-as-promised");
 import childProcess = require("child_process");
 import q = require("q");
 import path = require("path");
-import _ = require("underscore");
 import {ChildProcess} from "child_process";
 import {ServerOptions} from "../src/server";
-import decamelize = require("decamelize");
 import providerMock from "../test/integration/provider-mock";
+const decamelize = require("decamelize");
```

Comment:
isn't import and const the same thing in this case?

Reply:
No, `const` doesn't try to resolve the module whereas `import` does. e.g. in this case the following error appears when using `import`:

```sh
bin/pact-cli.spec.ts(9,29): error TS7016: Could not find a declaration file for module 'decamelize'. '/Users/mfellows/development/public/pact-node/node_modules/decamelize/index.js' implicitly has an 'any' type.
  Try `npm install @types/decamelize` if it exists or add a new declaration (.d.ts) file containing `declare module 'decamelize';`
```